src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { memo } from 'react';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  pathName: string,
+}
+
+export const TodoFilter = memo<Props>(({ pathName }) => (
+  <ul className="filters">
+    <li>
+      <Link
+        to="/"
+        className={classNames({
+          selected: pathName === '/',
+        })}
+      >
+        All
+      </Link>
```

Comment:
I suppose you could use `NavLink` so that you can apply `isActive` property
by doing so, you avoid passing `pathName`, it looks redundant
also, as mentioned, you could use `useParams`